SWLOR.Game.Server/Feature/StatusEffectDefinition/AuraStatusEffectDefinition.cs

```diff
@@ -53,6 +53,7 @@ private void Charge()
                 .RemoveAction((target, data) =>
                 {
                     RemoveEffectByTag(target, EffectTag);
+                    DelayCommand(0.5f,() => Stat.ApplyPlayerMovementRate(target));
```

Comment:
Why is a delay needed here?

Reply:
It was needed to make the movement apply but I'm just going to shelf this for now and revisit at another time and try to find a way to make it would without running a delay after every time you get exit aura.